package.json

```diff
@@ -582,7 +583,8 @@
   },
   "prettier": {
     "singleQuote": true,
-    "bracketSpacing": false
+    "bracketSpacing": false,
+    "endOfLine": "auto"
```

Comment:
lexical order?

Reply:
unless we can lint and automatically format it, I'm not interested in enforcing lexical order anywhere.  it's nice, but also IMO a nit.

Reply:
> unless we can lint and automatically format it, I'm not interested in enforcing lexical order anywhere. it's nice, but also IMO a nit.

Wholeheartedly disagree. We shouldn't _require_ lint and autoformat just to put items in alphabetical order, and not doing so just makes maintenance harder (requiring search rather than skim). There are only two properties in this file ('prettier' and 'yargs') that really _aren't_, and both can be trivially corrected.